Examples/MAX32690/Hello_World/README.md

```diff
@@ -27,11 +27,14 @@ Universal instructions on building, flashing, and debugging this project can be
 The Console UART of the device will output these messages:
 
 ```
+Hello class!
+Hello method!
 Hello World!
```

Comment:
Is this expected output correct for the non-CPP example?

Reply:
Thanks, fixed in https://github.com/Analog-Devices-MSDK/msdk/pull/619/commits/9e9a91c38b07aa42cbce6d9fc72081a856c422ae